remote/gitlab/client/project.go

```diff
@@ -1,17 +1,16 @@
 package client
 
 import (
+	b64 "encoding/base64"
```

Comment:
remove alias and use `base64` package name below

Reply:
Done.